sihl/src/core_command.ml

```diff
@@ -8,9 +8,10 @@ type t =
   { name : string
   ; usage : string option
   ; description : string
-  ; dependencies : Core_lifecycle.lifecycle list
-  ; fn : string list -> unit option Lwt.t
+  ; dependencies : Core_lifecycle.lifecycle list [@opaque]
+  ; fn : string list -> unit option Lwt.t [@opaque]
   }
+[@@deriving show]
```

Comment:
`eq` is missing here on purpose?

Reply:
Yes, it's not clear how to define `eq` with functions

Reply:
For the `Core_schedule.t` you did `fn : unit -> unit Lwt.t [@equal fun _ _ -> true]`, is this not applicable here too?

Reply:
That's actually a fair point!